virttest/qemu_devices/qdevices.py

```diff
@@ -1505,6 +1505,30 @@ def verify_hotplug(self, out, monitor):
         return False
 
 
+class QSEVObject(QObject):
```

Comment:
no need to introduce new objects, just use QObject class

Reply:
a device in the device tree
then the qemu cmd

Reply:
classes, e.g.
-object
-device

For the backends of the same object, eg.
-object sev-guest
-object tdx-guest
No need to derive from object